Inspicio/Models/Comment.cs

```diff
@@ -12,10 +13,14 @@ public class Comment
         public int CommentID { get; set; }
```

Comment:
CommentID doesn't match the same naming conventions as the other id's e.g OwnerId, ImageId etc..